Lib/pdb.py

```diff
@@ -1132,9 +1132,9 @@ def do_args(self, arg):
         """
         co = self.curframe.f_code
         dict = self.curframe_locals
-        n = co.co_argcount
-        if co.co_flags & 4: n = n+1
```

Comment:
Thanks for removing these evil magic numbers!

Reply:
Definitely the improvement Python needed ^^